### ISSUE TITLE
When running an example on the README.md command gets stuck (using .take)

### ISSUE THREAD

**Issue by ribanez7** [rossta/montrose#101]
Hi Ross,

I am working with the gem to see if it fits my needs. I have seen that when running this piece of code on the documentation, the console gets stuck and the return value is never reached:

```ruby
opts = Montrose::Recurrence.new(every: 10.minutes).to_h
Montrose::Recurrence.new(opts).take(3)
```

Do you have any idea?

![image](https://user-images.githubusercontent.com/10326293/43263530-cee71396-90e3-11e8-8840-8509d272e44b.png)

**Comment by rossta** [comment]
@ribanez7 Thanks for reporting this issue. This is a bug and I am working on the fix.

**Comment by ribanez7** [comment]
Thanks @rossta ! I have build the gem locally and it's fixed now. Will you upload it to rubygems? Thanks

**Comment by rossta** [comment]
@ribanez7 Just released `0.8.1` with the fix.
